lib/kafka/fetcher.rb

```diff
@@ -0,0 +1,178 @@
+require "kafka/fetch_operation"
+
+module Kafka
+  class Fetcher
+    MAX_QUEUE_SIZE = 100
```

Comment:
Could you please document that variable? does it describe the max ahead buffer size or max fetch size? What the queue is? Thanks! :)